lib/feedback.ml

```diff
@@ -64,7 +64,9 @@ let feedback ?id ?route what =
      route = Option.default !feedback_route route;
      id    = Option.default !feedback_id id;
   } in
-  Hashtbl.iter (fun _ f -> f m) feeders
+  let at_least_one_sink =
+    Hashtbl.fold (fun _ f _ -> f m; true) feeders false in
+  assert at_least_one_sink
```

Comment:
I don't support adding random asserts; at most, this should be warning. There are uses cases where no feedback handler is installed.

Reply:
Who will be responsible for displaying this warning? One of the non-installed feedback handler?

Let us be serious for a moment: if coqchk is trying to display an error message and there is no feedback handler, this should be a visible fatal error!

Reply:
IMHO, this should be handled by the proper entries in the test-suite to verify that `coqchk`works properly, not by a random assert on what is a coqchk bug.